src/torchcodec/decoders/_core/VideoDecoder.h

```diff
@@ -139,9 +139,7 @@ class VideoDecoder {
   void addVideoStream(
       int streamIndex,
       const VideoStreamOptions& videoStreamOptions = VideoStreamOptions());
-  void addAudioStreamDecoder(
-      int streamIndex,
-      const AudioStreamOptions& audioStreamOptions = AudioStreamOptions());
+  void addAudioStream(int streamIndex);
```

Comment:
I only just now realized that we've had `addAudioStreamDecoder()` declared in the .h all this time, with no implementation. 🤯